components/JsonMessageEditor.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useRef, MutableRefObject } from 'react'
+import JSONEditor from 'jsoneditor'
+import 'jsoneditor/dist/jsoneditor.css'
+
+export const JSONMessageEditor = ({
```

Comment:
Thoughts on using same naming convention as your other JSON related components? i.e. `JsonMessageEditor`

Reply:
That's good. Making it so.